ci-operator/step-registry/servicemesh/istio/snc/mapt/create/servicemesh-istio-snc-mapt-create-commands.sh

```
@@ -0,0 +1,145 @@
+    if [ $exit_code -eq 0 ] && ! echo "$output" | grep -qiE "(stderr|error|failed|exit status [1-9])"; then
+      echo "[SUCCESS] !!!! Successfully destroyed MAPT infrastructure during cleanup"
+    else
+      echo "[WARN] WARN MAPT destroy may have failed, but continuing with bucket cleanup"
+    fi
+
+    echo "[INFO] **** Emergency cleanup: Deleting S3 bucket: ${DYNAMIC_BUCKET_NAME}..."
+
+    aws s3 rm "s3://${DYNAMIC_BUCKET_NAME}" --recursive 2>/dev/null || true
+
+    aws s3api list-object-versions --bucket "${DYNAMIC_BUCKET_NAME}" --output json 2>/dev/null | \
+    jq -r '.Versions[]?, .DeleteMarkers[]? | "aws s3api delete-object --bucket '${DYNAMIC_BUCKET_NAME}' --key \"" + .Key + "\" --version-id " + .VersionId' | \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)